modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -37,6 +39,9 @@ Create a new application for the API Experience Hub portal in the identity provi
 |*Selected group(s)*|Enter the name of the group who must have access to the application.
 |===
 . Configure a custom groups claim filter that the application sends.
++
+If you are not using OpenID Connect, review the xref:enabling-sso-for-your-portal.adoc#before-you-begin[Before You Begin] section for the values that API Experience expects for claims and attributes and the documentation for the identity provider. 
+
```

Comment:
```suggestion
. Configure required claims that the application sends.
+
Review the xref:enabling-sso-for-your-portal.adoc#before-you-begin[Before You Begin] section to verify that your Identity Provider send the expected claims.
For example, for OpenID in Okta you can configure the groups claim by doing the following:
```

Reply:
should you review the Before You Begin section before the step in line 41 Configure required claims that the application sends.

Reply:
Thanks @nbattiato, I think it makes sense to move the step to review the Before you begin section before you configure the claims.